gui/wxpython/gui_core/wrap.py

```diff
@@ -144,7 +144,7 @@ def SetToolTip(self, tip):
                 wx.Window.UnsetToolTip(self)
             else:
                 wx.Window.SetToolTip(self, tipString=tip)
-        else:
+        else:  # noqa: PLR5501
```

Comment:
This could be probably changed too...

Reply:
Was a pattern that could be better simplifies

Reply:
Couldn't find a better way here. Visually similar, but not the same functions called.